src/azure-cli/azure/cli/command_modules/profile/_validators.py

```diff
@@ -3,35 +3,29 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import uuid
-
 from azure.cli.core.util import should_disable_connection_verify
 from knack.log import get_logger
 
 logger = get_logger(__name__)
 
 
-def _is_guid(guid):
-    try:
-        uuid.UUID(guid)
-        return True
-    except ValueError:
-        return False
-
-
 def validate_tenant(cmd, namespace):
     """
     Make sure tenant is a GUID. If domain name is provided, resolve to GUID.
     https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc#fetch-the-openid-connect-metadata-document
     """
-    if namespace.tenant is not None and not _is_guid(namespace.tenant):
+    from azure.cli.core.util import is_guid
+    if namespace.tenant is not None and not is_guid(namespace.tenant):
         import requests
         active_directory_endpoint = cmd.cli_ctx.cloud.endpoints.active_directory
         url = '{}/{}/.well-known/openid-configuration'.format(active_directory_endpoint, namespace.tenant)
-        metadata = requests.get(url, verify=not should_disable_connection_verify()).json()
+        response = requests.get(url, verify=not should_disable_connection_verify())
 
-        # Example issuer: https://sts.windows.net/72f988bf-86f1-41af-91ab-2d7cd011db47/
-        tenant_id = metadata['issuer'].split("/")[3]
+        if response.status_code != 200:
```

Comment:
Just want to confirm, is 200 the only valid value?

Reply:
I think so, but the doc [Fetch the OpenID Connect metadata document](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc#fetch-the-openid-connect-metadata-document) never gives a clear confirmation.